stable/consul/templates/consul-ingress.yaml

```diff
@@ -1,6 +1,7 @@
 {{- if .Values.uiIngress.enabled -}}
 {{- $releaseName := .Release.Name -}}
 {{- $servicePort := .Values.HttpPort -}}
+{{- $consulFullName := default .Chart.Name .Values.nameOverride -}}
```

Comment:
You should use the fullname partial. I'd suggest you create a variable for the complete service name and then use that below.

From the top of my head, this should work:
`{{- $serviceName := printf "%s-%s" (include "consul.fullname" .) ui -}}`